pyftpdlib/test/__init__.py

```diff
@@ -362,9 +362,11 @@ class ThreadedTestFTPd(ThreadWorker):
     shutdown_after = 10
     poll_interval = 0.001 if TRAVIS else 0.000001
 
-    def __init__(self, addr=None):
+    def __init__(self, addr=None, handler=None):
         super(ThreadedTestFTPd, self).__init__(
             poll_interval=self.poll_interval)
+        if handler:
+            self.handler = handler
```

Comment:
I prefer not to do this. You can simply `setattr` the handler later against this object